java/src/test/java/me/everything/inbloom/BloomFilterTest.java

```diff
@@ -9,6 +9,102 @@
  */
 public class BloomFilterTest extends TestCase {
 
+    public void testCreateFilterWithBadParameters()
+    {
+        try 
+        {
+            BloomFilter_mutated bf = new BloomFilter_mutated( -1, 0.01 );
```

Comment:
Done. Let me know if you see the changes. 
Good catch btw! This was my own mistake by not running the tests on the final copy. Lesson learned.